lib/blockchain.go

```diff
@@ -4906,7 +4906,17 @@ func (bc *Blockchain) CreateMaxSpend(
 		prevFeeAmountNanos := uint64(0)
 		for feeAmountNanos == 0 || feeAmountNanos != prevFeeAmountNanos {
 			prevFeeAmountNanos = feeAmountNanos
-			feeAmountNanos = _computeMaxTxV1Fee(txn, minFeeRateNanosPerKB)
+			if bc.BlockTip().Height >= bc.params.ForkHeights.ProofOfStake2ConsensusCutoverBlockHeight &&
+				feeEstimator != nil {
+				// TODO: replace MaxBasisPoints and 100*100 with variables configured by flags.
+				feeAmountNanos, err = feeEstimator.EstimateFee(txn, MaxBasisPoints, 100*100,
```

Comment:
May as well add a PR that creates a const for the 100*100 now, and then switch it to a flag later. This way, we don't have the 100*100 sprinkled across the codebase now

Reply:
We'll just use `MaxBasisPoints` for now.